### PR TITLE
Add Google Health COVID notebook, and change sandbox to research in t…

### PR DESCRIPTION
**Pull request by tjann**
…he Colab URLs.

### CODE REVIEW

File: tutorials.md [hunk 1]
```diff
@@ -17,19 +17,23 @@ Python.
 
 -   **Getting Started: Analyzing Census Data with Data Commons**
 
-    [Google Colab](https://colab.sandbox.google.com/drive/1wYohDirOgVxvmL0d-oJRWdD6AXfAX_w1){:target="_blank"}, [Github](https://github.com/datacommonsorg/api-python/blob/master/notebooks/analyzing_census_data.ipynb){:target="_blank"}
+    [Google Colab](https://colab.research.google.com/drive/1wYohDirOgVxvmL0d-oJRWdD6AXfAX_w1){:target="_blank"}, [Github](https://github.com/datacommonsorg/api-python/blob/master/notebooks/analyzing_census_data.ipynb){:target="_blank"}
+
+-   **COVID-19 Feature Exploration Analysis with Data Commons (by Google Health)**
```
Comment (tjann) [inline tutorials.md:22]: I am soliciting opinions on the title. Notice how all the other ones have
"Getting Started:" or "Case Study:"

**Comment by tjann** [comment]
Should submit https://github.com/datacommonsorg/api-python/pull/160 before this PR.

File: tutorials.md (hunk 1 above)
Comment (pradh) [reply tutorials.md:22]: I like consistent naming :) How about:

  **Case Study: COVID-19 Feature Exploration Analysis (by Google Health)**

"with Data Commons" I feel is redundant and the ones below don't have it.

**Review by pradh — APPROVED** [review]

**Review by beets — APPROVED** [review]
very cool to add this notebook!

File: tutorials.md (hunk 1 above)
Comment (tjann) [reply tutorials.md]: Done, thanks for the opinion!

**Pull request merged**